### PR TITLE
feat: Allow esbuild 0.24

### PR DESCRIPTION
**Pull request by danilofuchs**

### CODE REVIEW

File: esbuild-node-externals/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ When bundling with Esbuild for the backend by default it will try to bundle all
 
 ## Installation
 
-This plugin requires minimum **Node.js 12**, and **Esbuild 0.12+**.
+This plugin requires minimum **Node.js 18**, and **Esbuild 0.21+**.
```
Comment (pradel) [inline esbuild-node-externals/README.md:12]: why did you change the minimum esbuild version here? It's didn't change in the package.json and as there is no breaking changes in these releases for our plugin it should be fine no?

**Review by pradel — CHANGES_REQUESTED** [review]

**Comment by antongolub** [comment]
Hmm... I think, if the plugin is still still [compatible with esbuild@0.12.x](https://github.com/pradel/esbuild-node-externals/pull/65/checks#step:9:47), we have no reason to restrict nodejs version.

**Comment by danilofuchs** [comment]
> Hmm... I think, if the plugin is still still [compatible with esbuild@0.12.x](https://github.com/pradel/esbuild-node-externals/pull/65/checks#step:9:47), we have no reason to restrict nodejs version.

My mistake! I wrongly read `esbuild 0.21 - 0.23` instead of `esbuild 0.12 - 0.23` in the package.json and assumed the minimum node version was out of date

**Comment by danilofuchs** [comment]
Closed in favour of https://github.com/pradel/esbuild-node-externals/pull/65

**Pull request closed without merging**